### ISSUE TITLE
[Backend] Add proper logging to all backend services

### ISSUE THREAD

**Issue by emilwihlander** [Dsek-LTH/member-page#231]
## Short description of the feature
Add proper logging using winston to all backend services, using correlation Ids and maybe logz.io(?)

Do this issue depend on another issue? Write Depends on #Issue_number here

[ ] Relates to frontend
[X] Relates to the backend
[ ] Requires modification of GraphQL
[ ] Requires modification of Database
[ ] Requires creating a new service
